lib/iris/tests/unit/quickplot/test_plot.py

```diff
@@ -29,5 +30,29 @@ def test_xaxis_labels(self):
         self.assertBoundsTickLabels("xaxis")
 
 
+class TestAxisLabels(tests.GraphicsTest):
+    def test_xy_cube(self):
+        c = simple_2d()[:, 0]
```

Comment:
54f952a4f1af5d5d8e5a9b5cc3c52b01fd83ee10
I tried `simple_1d` and the only failures were those one would expect since the coordinate names were different. Corrected the references in the 2 tests and all ran fine.